shuffle/sample-app/src/main.rs

```diff
@@ -38,9 +53,10 @@ fn main() -> Result<()> {
     let root_account_key = load_key(root_key_path);
```

Comment:
This `fn main` method is over 100 lines long. Can we break it up into methods to
1. improve readability with self-documented code via the method name
2. modularize the work so it's easier for someone (like me) to come behind and refactor/incorporate this into the new cli